lightwallet-client-lib/src/main/java/co/electriccoin/lightwallet/client/internal/CoroutineLightWalletClientImpl.kt

```diff
@@ -130,7 +128,7 @@ internal class CoroutineLightWalletClientImpl private constructor(
             "${Constants.ILLEGAL_ARGUMENT_EXCEPTION_MESSAGE} address: $tAddress." // NON-NLS
         }
         return requireChannel().createStub().getAddressUtxosStream(
-            Service.GetAddressUtxosArg.newBuilder().setAddress(tAddress)
+            Service.GetAddressUtxosArg.newBuilder().setAddresses(Constants.TRANSPARENT_ADDRESS_INDEX, tAddress)
```

Comment:
this is bringing up some questions on how the android sdk is requesting UTXOs. It has nothing to do with the API itself, but with the caller.

please double check that the SDK requests UTXO for all existing derived addresses. this means calling the rust API to get those addresses and call a function in this lightwalletd client that accepts many addresses as input (probably an array or sequence of them)

Reply:
Thanks for pointing this out. I just quickly looked at it. If I understand it right, the UTXOs are refreshed e.g. after scan is complete [here](https://github.com/zcash/zcash-android-wallet-sdk/blob/01da937fdcf2d5d70686d5617ecad623f138f685/sdk-lib/src/main/java/cash/z/ecc/android/sdk/SdkSynchronizer.kt#L405) and the [getTransparentAddress](https://github.com/zcash/zcash-android-wallet-sdk/blob/01da937fdcf2d5d70686d5617ecad623f138f685/sdk-lib/src/main/java/cash/z/ecc/android/sdk/Synchronizer.kt#L188) works with `Account.DEFAULT` for now. Moreover, until now we worked with the simple `getAddress` here in networking. So I assume we are checking the only one address, but I might we wrong. So do you want me to file an issue, so we can check it more in details and we're not blocked in this PR?

Reply:
Yes this doesn't block this PR and can be worked in a subsequent PR but rather urgently if you ask me

Reply:
Yep, here I just filed the issue for it https://github.com/zcash/zcash-android-wallet-sdk/issues/904. Please, add any additional details, if needed.